src/test/testWebpackSourceMaps.ts

```diff
@@ -26,43 +27,41 @@ describe('Webpack sourcemaps: The debugger', function() {
 		}
 	});
 
-	for (let devtool of [
-		'cheap-eval-source-map', 'cheap-source-map', 'cheap-module-eval-source-map', 'inline-source-map',
-		'cheap-module-source-map' , 'eval-source-map' , 'source-map' , 'nosources-source-map'
-	]) {
+	for (const devtool of [
+		'source-map', 'inline-source-map', 'nosources-source-map', 'inline-nosources-source-map',
+		'eval-source-map', 'eval-cheap-source-map', 'eval-cheap-module-source-map',
+		'eval-nosources-source-map', 'eval-nosources-cheap-source-map', 'eval-nosources-cheap-module-source-map',
```

Comment:
Could we put one item per line so it's easier to see changes?
```suggestion
	for (const devtool of [
		'source-map', 
		'inline-source-map', 
		'nosources-source-map', 
		'inline-nosources-source-map',
		'eval-source-map', 
		'eval-cheap-source-map', 
		'eval-cheap-module-source-map',
		'eval-nosources-source-map', 
		'eval-nosources-cheap-source-map', 
		'eval-nosources-cheap-module-source-map',
```